example/config.yml

```diff
@@ -15,6 +15,7 @@ collections: # A list of collections the CMS should be able to edit
       - {label: "Publish Date", name: "date", widget: "datetime", format: "YYYY-MM-DD hh:mma"}
       - {label: "Cover Image", name: "image", widget: "image", required: false, tagname: ""}
       - {label: "Body", name: "body", widget: "markdown"}
+      - {label: "Some Field", name: "somefield", widget: "string", required: false} # Testing empty field
```

Comment:
I just removed the new test content that was in there... it was there for the other PR I did with the space in the image file name issue, and I just reused it to test this one. It's gone now.